### PR TITLE
Pg crypt

### PR DESCRIPTION
**Pull request by rachelriv**
- encrypt passwords
- give processing folks everything they need

### CODE REVIEW

File: database/postgresSetup.sql [hunk 1]
```diff
@@ -32,7 +32,7 @@ CREATE TABLE IF NOT EXISTS events(
       user_id int REFERENCES users,
       PRIMARY KEY (feed_id, user_id));
 
-INSERT INTO users(username, password) values ('foo', 'bar');
+INSERT INTO users(username, password) values ('foo', crypt('bar', gen_salt('md5')));
```
Comment (rtoal) [inline database/postgresSetup.sql:35]: Part of a test suite?

**Comment by juansc** [comment]
LGTM

File: database/postgresSetup.sql (hunk 1 above)
Comment (rachelriv) [inline database/postgresSetup.sql:35]: @jtfennell suggested that we start hashing passwords, which is a great idea. This script populates the DB with sample data. 

I'm a little confused by your question.

Comment (rtoal) [inline database/postgresSetup.sql:35]: Right, I know what encrypting and salting are for but the file is just called `database/postgresSetup.sql` and it isn't clear from context whether populating the database with mock data is done during testing or app startup. If this is the real database it shouldn't start with anything in it. (Attackers looking at this project -- it's open source after all -- can see there is a foo/bar user and easily break into a Grapevine installation.) If the mock data is created only during testing then that's okay.

Comment (rachelriv) [inline database/postgresSetup.sql:35]: Oh yes, it is just used for testing. Sorry I misunderstood!

Comment (rachelriv) [inline database/postgresSetup.sql:35]: We will rename the file so that is clearer. :smile:

**Pull request merged**